<launch>
  <include file="$(find uvc_camera)/launch/stereo_image_proc.launch" />

  <node name="cmvision" pkg="cmvision" type="cmvision_node"
        output="screen">
    <param name="color_file" type="string" value="$(find cmvision)/config/colors.txt" />
    <param name="image_topic" type="string" value="/stereo/right/image_rect_color"/>
    <param name="color_radius_pix" type="double" value="40.0"/>
    <param name="spatial_radius_pix" type="double" value="2.0"/>
    <param name="debug_on" type="bool" value="false"/>
    <param name="color_cal_on" type="bool" value="false"/>
    <param name="mean_shift_on" type="bool" value="false"/>
  </node>
</launch>
